test/test_jit.py

```diff
@@ -3350,6 +3350,18 @@ def test_view_shape_prop(a):
         real_outs = cu.test_view_shape_prop(*inputs)
         self.assertEqual(real_outs, outputs)
 
+    def test_view_listconstruct_shape_prop(self):
+        def fn(x):
+            B = x.size(0)
+            C = x.size(1)
+            T = x.size(2)
+            return x.view(T, B, C)
+
+        x = torch.randn(3, 1, 5, requires_grad=True)
+        graph = torch.jit.script(fn).graph
+        torch._C._jit_pass_complete_shape_analysis(graph, (x,), False)
+        self.assertExpectedGraph(graph)
```

Comment:
Can you please just check that `next(graph.outputs()).type().kind() != 'DynamicType'`? We don't need more expects for such simple things